data/OR/incentives.json

```diff
@@ -0,0 +1,962 @@
+    "item": "rooftop_solar_installation",
+    "program": "or_pacificPower_solarWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1,
+      "unit": "watt",
+      "maximum": 6000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$0.90/watt for solar. $6,000 maximum per home. For income-qualified households. System must be installed by an approved solar trade ally."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-15",
+    "authority_type": "utility",
+    "authority": "or-portland-general-electric",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "battery_storage_installation",
+    "program": "or_portlandGeneralElectric_solarWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 750,
+      "unit": "kilowatt",
+      "maximum": 10000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750/kWh for battery storage. $10,000 maximum per home. For income-qualified households. System must be installed by an approved solar trade ally."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-16",
+    "authority_type": "utility",
+    "authority": "or-pacific-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "battery_storage_installation",
+    "program": "or_pacificPower_solarWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 750,
+      "unit": "kilowatt",
+      "maximum": 10000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750/kWh for battery storage. $10,000 maximum per home. For income-qualified households. System must be installed by an approved solar trade ally."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-17",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 100
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$100 off qualifying Google Nest and ecobee Smart Thermostats."
+    }
+  },
+  {
+    "id": "OR-18",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$1 per sqft for high efficiency windows. Windows must replace existing windows, glass doors or skylights. U-value must be between 0.27 – 0.23."
```

Comment:
optional: I don't think I've seen `sqft` before – take a look through our other incentives to see if we have a different way of representing this, e.g. `sq. ft`.

If so, change the others like it.